README.md

```diff
@@ -33,12 +33,12 @@ You can generate `std4`'s documentation with
 
 ```text
 # if you're generating documentation for the first time
-> lake -Kdoc=on update
+> lake -R -Kdoc=on update
 ...
 # actually generate the documentation
-> lake -Kdoc=on build Std:docs
+> lake -R -Kdoc=on build Std:docs
```

Comment:
Why is `-R` needed? Please include that in the PR description.

Reply:
The `-R` flag generates configuration files that seems to be needed by [doc-gen4](https://github.com/leanprover/doc-gen4), as documented on the README. Without the flag, the docs didn't build for me. I also found [this interaction](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/how.20to.20generate.20html.20documentation.20for.20new.20lean.20file/near/419851869) on Zulip, which seems to suggest the `-R` flag is required. Adding it fixed my issue.